activities/Abecedarium.activity/play.js

```diff
@@ -52,6 +64,9 @@ enyo.kind({
 			this.doGame(this.themeButton);
 		else
 			this.displayButtons();
+			
+		if (this.gamecount === entriesByGame)
+			this.showGameFinished();
```

Comment:
Use bracket around instructions in if/else

Reply:
@llaske pre-existing code in lines above did not use brackets so I intentionally left them for uniformity in style. Will change it.